docs/debuginfod.md

```diff
@@ -0,0 +1,35 @@
+# debuginfod client
+
+In order to symbolize ingested profiles, Parca needs to have debug information
+for the binaries that are being profiled. Debug information
+can typically comprises ELF object files, DWARF debug data, and source code
+required for symbolization. However, sometimes, application packages distributed
+by various Linux distros strip away debug information to minimize the size of
+the binaries. Thankfully, there are public vendor distributed servers which make distro
+specific debug info files available to the community.
+
+debuginfod is an HTTP file server that serves debug information to clients
+based on the build IDs of the packages. Parca has a custom debuginfod client for querying upstream
```

Comment:
the build IDs are of the binaries

I think here it might be useful to quickly show people what the build ID is with the `file` command

Reply:
oops, good catch, thanks!